dotnet-desktop-guide/net/winforms/data/multiple-controls-bound-to-data-source-synchronized.md

```diff
@@ -0,0 +1,81 @@
+
+### Change notification for simple binding
+
+For simple binding, business objects must provide change notification when the value of a bound property changes. You can provide change notification by exposing a PropertyNameChanged event for each property of your business object. It also requires binding the business object to controls with the <xref:System.Windows.Forms.BindingSource> or the preferred method in which your business object implements the <xref:System.ComponentModel.INotifyPropertyChanged> interface and raises a <xref:System.ComponentModel.INotifyPropertyChanged.PropertyChanged> event when the value of a property changes. When you use objects that implement the `INotifyPropertyChanged` interface, you don't have to use the `BindingSource` to bind the object to a control. But using the `BindingSource` is recommended.
+
+### Change notification for list-based binding
+
+Windows Forms depends on a bound list to provide *property change* and *list change* information to bound controls. The *property change* is a list item property value change and *list change* is an item deleted or added to the list. Therefore, lists used for data binding must implement the <xref:System.ComponentModel.IBindingList>, which provides both types of change notification. The <xref:System.ComponentModel.BindingList%601> is a generic implementation of `IBindingList` and is designed for use with Windows Forms data binding. You can create a `BindingList` that contains a business object type that implements <xref:System.ComponentModel.INotifyPropertyChanged> and the list will automatically convert the <xref:System.ComponentModel.INotifyPropertyChanged.PropertyChanged> events to <xref:System.ComponentModel.IBindingList.ListChanged> events. If the bound list isn't an `IBindingList`, you must bind the list of objects to Windows Forms controls by using the <xref:System.Windows.Forms.BindingSource> component. The `BindingSource` component will provide property-to-list conversion similar to that of the `BindingList`. For more information, see [How to: Raise Change Notifications Using a BindingSource and the INotifyPropertyChanged Interface](/dotnet/desktop/winforms/controls/raise-change-notifications--bindingsource?view=netframeworkdesktop-4.8&preserve-view=true).
+
+### Change notification for custom controls
+
+Finally, from the control side you must expose a *PropertyName*Changed event for each property designed to be bound to data. The changes to the control property are then propagated to the bound data source. For more information, see [How to: Apply the PropertyNameChanged Pattern](/dotnet/desktop/winforms/how-to-apply-the-propertynamechanged-pattern?view=netframeworkdesktop-4.8&preserve-view=true)
```

Comment:
space needed after *PropertyName*

Reply:
The method name format doesn't have space.